.env.dist

```diff
@@ -8,3 +8,4 @@ MYSQL_PASSWORD=wordpress
 WP_VERSION=5.3.2
 WP_DB_USER=wordpress
 WP_DB_PASSWORD=wordpress
+WP_HTTP_PORT=8088
```

Comment:
Why would we want to make this configurable? We would need to change documentation and console output

Reply:
As a developer I’d sometimes want to have the ability to switch ports for individual projects so I don’t have to keep track of shutting down containers and run one site at a time. 

However, we provide the 8088 default and you basically never need to adjust it.

Reply:
MySQL port `3306` will collide and you can't run two projects at the same time anyhow so this would be an unnecessary change if that is the reason to add it.

Reply:
Yes that’s true. Didn’t think of that. I guess asking to customize the mysql port too would be a reach 😄